### ISSUE TITLE
loss Function

### ISSUE THREAD

**Issue by Mrlong12** [xw-hu/DSC#16]
请问胡老师，你的损失函数有pytorch版本吗？

**Comment by xw-hu** [comment]
A PyTorch version is avaliable at https://github.com/stevewongv/DSC-PyTorch implementated by Tianyu Wang.
